_std/defines/obj.dm

```diff
@@ -11,6 +11,8 @@
 #define HAS_DIRECTIONAL_BLOCKING (1<<3)
 /// prevents ghost critter interaction. On obj so it can cover machinery, items etc...
 #define NO_GHOSTCRITTER (1<<4)
+/// Item treats locker movement as being moved itself (calls OnMove())
+#define TRACK_STORAGE_MOVEMENT		(1<<5)
```

Comment:
Why not just track area exited or something, signal

Reply:
i wasn't aware that method existed, ill swap to that shortly